symbols.xml

```diff
@@ -1280,6 +1280,9 @@
 
         generated vtables
 
+        <vtable-address name='KeybindingScreen' value='0xf6b624'/>
+        <vtable-address name='MacroScreenLoad' value='0xf6b650'/>
+        <vtable-address name='MacroScreenSave' value='0xf6b67c'/>
```

Comment:
Right; I had found the angavrilov fork first, and failed to check the network before using it.